_data/consulting.yaml

```
@@ -0,0 +1,19 @@
+---
+- name: Ally Skills
+  url: /consulting/ally-skills.html
+  description:  |
+        Multiple OLS community members are trained Ally skill facilitators. 
+        We offer ad-hoc in-house workshops (starting at £2,000 GBP for universities) 
+        as well as anyone-may-register workshops...
+
+- name: Accessibilty Consulting
```

```suggestion
- name: Video call accessibility
```
Typo fix + narrowing down the scope - I don't want to assert we know all of accessibility!

---

Oops. Thank you!

And I know what you mean.